x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/lens_attributes.ts

```diff
@@ -572,7 +616,7 @@ export class LensAttributes {
       layers[layerId] = {
         columnOrder: [
           `x-axis-column-${layerId}`,
-          ...(breakdown && sourceField !== USE_BREAK_DOWN_COLUMN
+          ...(breakdown && sourceField !== USE_BREAK_DOWN_COLUMN && breakdown !== 'percentile'
```

Comment:
maybe `percentile` into a constant.

Reply:
Oooh, I did. I just missed this one. Thanks for catching it!